pkg/controller/user-cluster-controller-manager/resources/controller.go

```diff
@@ -20,6 +20,7 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	v1 "k8c.io/kubermatic/v2/pkg/crd/kubermatic/v1"
```

Comment:
This import should go down with other `k8c.io` imports. Also, avoid the `v1` import alias. We're using `kubermaticv1` as the import alias in other places.